realm/realm-jni/src/io_realm_internal_Util.cpp

```diff
@@ -78,7 +80,9 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_Util_nativeGetMemUsage(JNIEnv*, j
 JNIEXPORT jstring JNICALL Java_io_realm_internal_Util_nativeGetTablePrefix(
     JNIEnv* env, jclass)
 {
-    return to_jstring(env, string(TABLE_PREFIX));
+    string table_prefix(TABLE_PREFIX);
```

Comment:
std:string is used in "internal_version" below. I am not sure we have a consensus on this? But at least being consistent in the same PR would be nice :)

I prefer std::string above string though.